docs/CompatibleSoftware.rst

```diff
@@ -19,63 +19,191 @@ application, please contact ocio-dev as well.
+
+- `<http://wiki.blender.org/index.php/Dev:Ref/Release_Notes/2.64>`__,
+
+- `<http://wiki.blender.org/index.php/Dev:Ref/Release_Notes/2.64/Color_Management>`__.
+
+
+C++
+***
+
+The core OpenColorIO API is available for use in C++. Minimal code examples are also available in the source code distribution. 
+
+Website: `<http://github.com/AcademySoftwareFoundation/OpenColorIO/tree/master/export/OpenColorIO>`__
+
+Documentation: 
+
+Of particular note are
+
+- `<https://github.com/AcademySoftwareFoundation/OpenColorIO/tree/master/src/apps/ocioconvert>`__
+- `<https://github.com/AcademySoftwareFoundation/OpenColorIO/tree/master/src/apps/ociodisplay>`__
+
+
+Clarisse (Isotropix)
+********************
+
+Website: `<https://www.isotropix.com/learn/tutorials/managing-looks-with-opencolorio-ocio>`__
+
+
+CryEngine (CryTek)
+*******************
+
+CryEngine is a real-time game engine, targeting applications in the motion-picture market. While we don't know many details about the CryEngine OpenColorIO integration, we're looking forward to learning more as information becomes available.
+
+Website: `<https://www.cryengine.com/>`__
+
+
+DJV
+***
+
+Website: `<https://darbyjohnston.github.io/DJV/>`__
+
+
+Gaffer
+******
+
+Gaffer is a node based application for use in CG and VFX production, with a particular focus on lighting and look development.
+
+Website: `<http://www.gafferhq.org/>`__
+
+
+Houdini (SideFX)
+****************
+
+Website: `<https://www.sidefx.com>`__
+
+Supported Version: >= 16
+
+Documentation:
+
+
+- `<https://www.sidefx.com/docs/houdini/io/ocio.html>`__
+
+- `<https://www.sidefx.com/filmtv/whats-new-h16/>`__
+
+
+Java (Beta)
+***********
+
+The OpenColorIO API is available for use in Java. This integration is currently considered a work in progress, and should not be relied upon for critical production work.
+
+Website: `<https://github.com/AcademySoftwareFoundation/OpenColorIO/tree/master/src/bindings/java>`__
+
+
+Katana (Foundry)
+****************
+
+Website: `<http://www.thefoundry.co.uk/products/katana>`__
+
+Documentation:
+
+- 2D Nodes: OCIODisplay, OCIOColorSpace, OCIOCDLTransform
+
+- Monitor Panel: Full OCIO Support
 
 
 Krita
 *****
 
-`2D Paint - Open Source <http://www.krita.org/>`__
+Krita now support OpenColorIO for image viewing, allowing for the accurate painting of float32/OpenEXR imagery.
+
+Website: `<http://www.krita.org/>`__
+
+Documentation :
+
+- See `krita.org <http://www.krita.org/item/113-krita-starts-supporting-opencolorio>`__ for details.
+
+
+Mari (Foundry)
+**************
+
+Mari 1.4v1+ ships with native support for OpenColorIO in their display toolbar.
+
+Website: `<http://www.thefoundry.co.uk/products/mari>`__
+
+Documentation:
+
+- A `video demonstration <http://vimeo.com/32909648>`__ of the Mari OCIO workflow.
+
+
+Maya (Autodesk)
+***************
+
+Autodesk Maya supports OCIO since version 2016 (I believe Autodesk SynColor color management system can read OCIO configurations?), May 2016.
```

Comment:
A Maya explanation is 'Autodesk Maya is a 3D computer animation, modeling, simulation, and rendering software.'
The sentence is 'Maya supports OCIO since version 2016.'

The web link to the Maya product is https://www.autodesk.com/products/maya/overview
The web link to the Color Management support in Maya is the right one.

Reply:
This link would be useful too: https://knowledge.autodesk.com/support/maya-lt/learn-explore/caas/CloudHelp/cloudhelp/2016/ENU/MayaLT/files/GUID-4AE9AABC-A900-4276-AFE4-F9DCD96BA03D-htm.html